modules/edit-modes/src/lib/measure-distance-mode.ts

```diff
@@ -1,137 +1,180 @@
-// @ts-ignore
 import turfDistance from '@turf/distance';
-// @ts-ignore
+import { FeatureCollection } from '../geojson-types';
+import { ClickEvent, PointerMoveEvent, ModeProps, GuideFeatureCollection, Tooltip } from '../types';
+import { getPickedEditHandle } from '../utils';
 import memoize from '../memoize';
-import {
-  ClickEvent,
-  PointerMoveEvent,
-  Tooltip,
-  ModeProps,
-  GuideFeatureCollection,
-  EditHandleFeature,
-} from '../types';
-import { FeatureCollection, Position } from '../geojson-types';
 import { GeoJsonEditMode } from './geojson-edit-mode';
 
-const DEFAULT_TOOLTIPS = [];
-
 export class MeasureDistanceMode extends GeoJsonEditMode {
-  startingPoint: Readonly<EditHandleFeature> | null | undefined = null;
-  endingPoint: Readonly<EditHandleFeature> | null | undefined = null;
-  endingPointLocked = false;
+  _isMeasuringSessionFinished = false;
```

Comment:
Changing convention from non-underscore to underscore? Good to stay consistent with existing conventions in a repo.

Reply:
Ups, I've got a bit confused with code style now, i see that you already use undescore style in the project (for example here https://github.com/uber/nebula.gl/blob/master/modules/edit-modes/src/lib/scale-mode.ts or here https://github.com/uber/nebula.gl/blob/master/modules/edit-modes/src/lib/snappable-mode.ts).